### PR TITLE
Tidy up dztech keyboard.c/h and readmes

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

DZtech cleanup part 4. This last one's a little more involved - I tweaked the LED flags for RGB matrix as after aligning the arrays to resemble the layout macro, they didn't seem to make much sense. Apart from that, mostly whitespace changes.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by yanfali** [comment]
Can we get this tested on actual hardware? I'm uncomfortable with these changes unless they've been tested by users, otherwise it should go into breaking

**Comment by fauxpark** [comment]
@yanfali I reverted the LED flags changes - should be OK to merge now.

File: keyboards/dztech/dz60rgb/readme.md [hunk 1]
```diff
@@ -4,9 +4,14 @@
...
-* Hardware Supported: DZtech dz60rgb, dz60rgb-ansi
-* Hardware Availability: [kbdfans](https://kbdfans.myshopify.com/)
+* Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)
+* Hardware Supported: DZtech DZ60RGB V1 and V2
+* Hardware Availability: [KBDfans](https://kbdfans.com/)
+
+There are two versions of the DZ60RGB. Please use the appropriate firmware for your board.
+
+V1: STM32F303 (Arm)
+V2: ATmega32U4 (AVR)
```
Comment (noroadsleft) [inline keyboards/dztech/dz60rgb/readme.md]: Might be good to specify here that the Arm MCUs take `.bin` firmware and the AVR MCUs take `.hex`.

File: keyboards/dztech/dz60rgb_ansi/readme.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
-* Hardware Supported: DZtech dz60rgb, dz60rgb-ansi
-* Hardware Availability: [kbdfans](https://kbdfans.myshopify.com/)
+* Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)
+* Hardware Supported: DZtech DZ60RGB-ANSI V1 and V2
+* Hardware Availability: [KBDfans](https://kbdfans.com/)
+
+There are two versions of the DZ60RGB-ANSI. Please use the appropriate firmware for your board.
+
+V1: STM32F303 (Arm)
+V2: ATmega32U4 (AVR)
```
Comment (noroadsleft) [inline keyboards/dztech/dz60rgb_ansi/readme.md]: As above.

:wink:

File: keyboards/dztech/dz60rgb_wkl/readme.md [hunk 3]
```diff
@@ -4,9 +4,14 @@
...
-* Hardware Supported: DZtech dz60rgb, dz60rgb-ansi
-* Hardware Availability: [kbdfans](https://kbdfans.myshopify.com/)
+* Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)
+* Hardware Supported: DZtech DZ60RGB-WKL V1 and V2
+* Hardware Availability: [KBDfans](https://kbdfans.com/)
+
+There are two versions of the DZ60RGB-WKL. Please use the appropriate firmware for your board.
+
+V1: STM32F303 (Arm)
+V2: ATmega32U4 (AVR)
```
Comment (noroadsleft) [inline keyboards/dztech/dz60rgb_wkl/readme.md]: As above.

File: keyboards/dztech/dz65rgb/readme.md [hunk 4]
```diff
@@ -1,10 +1,17 @@
 # DZ65RGB
 
+
+
```
Comment (noroadsleft) [inline keyboards/dztech/dz65rgb/readme.md]: Leaving space for a photo later?

File: keyboards/dztech/dz65rgb/readme.md [hunk 5]
```diff
@@ -1,10 +1,17 @@
...
-* Hardware Supported: [DZtech](http://keyboarddiy.taobao.com)
-* Hardware Availability: [kbdfans](https://kbdfans.myshopify.com/)
+* Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)
+* Hardware Supported: DZ65RGB V1 and V2
+* Hardware Availability: [KBDfans](https://kbdfans.com/)
+
+There are two versions of the DZ65RGB. Please use the appropriate firmware for your board.
+
+V1: STM32F303 (Arm)
+V2: ATmega32U4 (AVR)
```
Comment (noroadsleft) [inline keyboards/dztech/dz65rgb/readme.md]: As above.

File: keyboards/dztech/dz65rgb/readme.md (hunk 4 above)
Comment (fauxpark) [reply keyboards/dztech/dz65rgb/readme.md]: Oops, I added that to compare between the four readmes in their separate tabs 😄

File: keyboards/dztech/dz60rgb/readme.md (hunk 1 above)
Comment (fauxpark) [reply keyboards/dztech/dz60rgb/readme.md]: Good point, done.

**Pull request merged**